app/src/main/res/drawable/splashscreen_legacy.xml

```diff
@@ -0,0 +1,17 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
The header is missing, it's because of a weird Git diff.

It's added here : #228